doc/changelog/04-tactics/12986-master+ordering-notation-by-precision.rst

```diff
@@ -0,0 +1,5 @@
+- **Changed:**
+  Use of notations for printing now (experimentally) gives preference
+  to notations which match a larger part of the term to abbreviate.
```

Comment:
If you want to insist on the experimental character, then you should also insist on what this means for this to be experimental. Since the idea is to receive feedback, I guess you could add the following:
```suggestion
  Use of notations for printing now (experimentally) gives preference
  to notations which match a larger part of the term to abbreviate.
  Please report any issue you encounter with this new printing strategy
```

Reply:
Agreed.